SetupAndDeployment.md

```diff
@@ -37,56 +37,54 @@ By the end of this module, you should be able to:
 
 ## Software Requirements
 
+- All the required software are available in google Drive, Use only those software
+
+`https://drive.google.com/drive/folders/11MGGSnwJVd1br2L3LeQQPzR4KgkiJ-cd `
```

Comment:
This is a great addition to the workshop content as this will help developers continue working with the content outside of an event.